docs/v1/P4Runtime-Spec.mdk

```diff
@@ -686,10 +686,11 @@ follows:
   for each pair of (`device_id`, `role_id`) at any point of time.
 
 * The streaming channel between the controller and the server defines the
-  liveness of the controller session. The controller is considered "offline" or
-  "dead" as soon as its corresponding stream channel to the switch is
-  broken, in which case the P4Runtime server quickly sets one of the slave
-  controllers with the highest `election_id` as master.
+  liveness of the controller session. The controller is considered “offline” or
+  “dead” as soon as its corresponding stream channel to the switch is broken, in
+  which case the P4Runtime server will be without a master, until one of the
```

Comment:
My only comment here is on the wording of "until one of the slave controllers".  The intended meaning is that any client that sends a successful master arbitration update message with a greater election_id than that of the previous master, yes?  As this is worded, it sounds like you may intend to restrict that to one of the _existing_ clients at the time the previous master died, but I don't think you intended any such restriction.

Perhaps something like the following?

Replace "until one of the slave controllers increases its election_id to a value higher than"

with: "until a client sends a successful `MasterArbitrationUpdate` message with an `election_id` value higher than"

Reply:
Done. I change "higher" to "greater or equal"

Reply:
I thought "or equal" was perhaps maybe wrong, and a new master must always have a strictly greater electon_id than the old master?

Reply:
I think the "or equal" is important. I've commented on this issue in a later comment.